Org Protocol Handler.app/Contents/Resources/parse.py

```diff
@@ -1,25 +1,38 @@
 #!/usr/bin/env python
 
+import ConfigParser
+import os
+import subprocess
 import sys
-import urlparse
 import urllib
-import subprocess
-import os
-import ConfigParser
+import urlparse
 
 
-def get_emacsclient_path():
-    """Get the configured path to `emacsclient`, or the default."""
+def read_config():
```

Comment:
Since this now returns the concatenated command, which is its actual purpose in life, the function should be named something like `read_emacsclient_command`.

`read_config` makes it sound like it's a pure side-effect function. It is confusing to mix the word "config" and variable names like "cmd" here with the current name.

Reply:
Agreed.  Also addressed in https://github.com/aaronbieber/org-protocol-handler/pull/5/commits/6110fe10832032d8efc5e2a815ac8caacc54981f